portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -109,6 +115,7 @@ $.extend(true, doaj, {
             this.title = edges.getParam(params.title, "");
 
             this.titleBar = edges.getParam(params.titleBar, true);
+            this.resultsOnly = edges.getParam(params.resultsOnly, false);
```

Comment:
Is this now necessary, if the FQWidget has its own template?

Reply:
Answering for myself - this template is not used with this arg, so technically it can come out.